ibm/resource_ibm_push_notification_chrome.go

```diff
@@ -0,0 +1,112 @@
+package ibm
+
+import (
+	"fmt"
+
+	"github.com/IBM/push-notifications-go-sdk/pushservicev1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceIBMPNApplicationChrome() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceApplicationChromeRead,
+		Create: resourceApplicationChromeCreate,
+		Update: resourceApplicationChromeUpdate,
+		Delete: resourceApplicationChromeDelete,
+
+		Schema: map[string]*schema.Schema{
+			"application_id": {
+				Type:        schema.TypeString,
```

Comment:
since application_id cann't be updated add ForceNew : true

Reply:
done